tests/foreman/ui/test_reporttemplates.py

```diff
@@ -396,23 +149,112 @@ def test_positive_e2e_crud(session, module_org, module_loc):
             'organizations.resources.assigned': [module_org.name],
             'locations.resources.assigned': [module_loc.name],
         })
-        assert session.reporttemplate.search(name)[0]['Name'] == name
-        pt = session.reporttemplate.read(name)
-        assert pt['template']['name'] == name
-        assert pt['template']['default'] is True
-        assert pt['template']['template_editor']['editor'] == content
-        assert pt['inputs'][0]['name'] == input_name
-        assert pt['inputs'][0]['required'] is True
-        assert pt['inputs'][0]['input_type'] == 'User input'
-        assert pt['type']['snippet'] is False
-        assert pt['locations']['resources']['assigned'][0] == module_loc.name
-        assert pt['organizations']['resources']['assigned'][0] == module_org.name
+        # also check other properties by API once BZ1702676 gets fixed
+        rt = entities.ReportTemplate().search(
```

Comment:
BZ is already ON_QA, anyway I am not sure why did you do search using API and then read through UI (which doing search through UI). That just doesn't make any sense